### PR TITLE
Suse and better windows

### PR DESCRIPTION
**Pull request by jcalonsoh**
### Description

Enable to install on suse 12+ and correct regex call on creating service on windows os

### Issues Resolved

Installing on windows server node 2012R2

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by jcalonsoh** [comment]
Original PR https://github.com/chef-cookbooks/push-jobs/pull/116

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+/.delivery/ merge=ours
```
Comment (iennae) [inline .gitattributes]: Can you put this in .git/info/attributes so it doesn't need to be committed with the project? Thanks!

File: .travis.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ services: docker
 
 env:
   matrix:
-  - INSTANCE=default-debian-7
+  # - INSTANCE=default-debian-7
```
Comment (iennae) [inline .travis.yml]: Any particular reason why you are commenting out debian-7 and centos-6? We like to have the tests running to expose any problems

File: .gitattributes (hunk 1 above)
Comment (jcalonsoh) [inline .gitattributes]: OK, I will do that

File: .travis.yml (hunk 2 above)
Comment (jcalonsoh) [inline .travis.yml]: Because it's asking a log template, I will do some more testing on that

**Comment by tas50** [comment]
While technically these distros aren't supported I think this is a good move

**Pull request merged**